WalletWasabi.Tests/UnitTests/QrDecode/QrCodeDecodingTests.cs

```diff
@@ -0,0 +1,81 @@
+using Xunit;
+using OpenCvSharp;
+using System.IO;
+using WalletWasabi.Helpers;
+
+namespace WalletWasabi.Tests.UnitTests.QrDecode
+{
+	public class QrCodeDecodingTests
+	{
+		private readonly string _commonPartialPath = Path.Combine(EnvironmentHelpers.GetFullBaseDirectory(), "UnitTests", "QrDecode", "QrResources");
+
+		[Fact]
+		public void GetCorrectAddressFromImages()
+		{
+			using QRCodeDetector decoder = new();
+			string expectedAddress = "tb1ql27ya3gufs5h0ptgjhjd0tm52fq6q0xrav7xza";
+			string otherExpectedAddress = "tb1qfas0k9rn8daqggu7wzp2yne9qdd5fr5wf2u478";
+
+			string path = Path.Combine(_commonPartialPath, "AddressTest1.png");
+			var qrImage = new Mat(path);
```

Comment:
Mat is disposable, so 
```suggestion
			using Mat qrImage = new(path);
```